.github/CODEOWNERS

```diff
@@ -39,22 +44,22 @@ scipy/stats/_resampling.py  @mdhaber
 scipy/stats/*sobol*  @tupui
 scipy/stats/_sensitivity_analysis.py  @tupui
 scipy/stats/_survival.py  @tupui @mdhaber
-scipy/stats/.unuran/*  @tirthasheshpatel
+scipy/stats/.unuran/  @tirthasheshpatel
 
 # Testing infrastructure
-ci/*  @larsoner @andyfaff
+ci/  @larsoner @andyfaff
 tools/refguide_check.py  @ev-br
-tools/*  @larsoner @rgommers
+tools/  @larsoner @rgommers
```

Comment:
I think this will override `tools/generate_requirements.py` above, I guess you could switch the order, but it seems not worth changing until the next round of changes to the file really, since the effect is only microscopically different.

Overall, based on the GitHub docs you linked, this looks way more correct than before.